app/models/user.rb

```diff
@@ -12,4 +12,16 @@ class User < ActiveRecord::Base
   def current_commitment
     commitments.first
   end
+
+  def stats
+    total_distance = routes.sum(:mileage)
+    total_prayers = Prayer.where(user_id: id).count
```

Comment:
This is correct, but you can also do it using a `has_many :prayers` association. Then you can just do `prayers.count`, which is a more ActiveRecord-y way of doing it.
It'll be just like how you're calling `routes` in the line above this.